sotodlib/hwp/update_hwp_angle.py

```diff
@@ -0,0 +1,517 @@
+import os
+import numpy as np
+import scipy.interpolate
+import so3g
+from spt3g import core
+import argparse
+
+class update_hwp_angle(): 
+    
+    def __init__(self, debug=False):
+
+        """
+        Class to manage a L2 HK data into HWP angle g3.
+
+        Args
+        -----
+            debug: bool
+                bit for dubug mode
+        """
+
+        
+        self._debug = debug
```

Comment:
instead of a setting like this we want to use a logger from the python package `logging`. You can look at `sotodlib/__init__.py` or `sotodlib/io/load_smurf.py` for examples. Instead you can use the debug flag to set the level of the logger.

Reply:
Implemented logging.